static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -121,7 +121,7 @@ export declare namespace TraceTree {
     };
   }
 
-  interface ChildrenAutogroup {
+  interface ChildrenAutogroup extends RawSpanType {
```

Comment:
Why?

Reply:
It's true, we spread the head node's value when we initialize the Auto-Grouped node [link](https://github.com/getsentry/sentry/blob/master/static/app/views/performance/newTraceDetails/traceTree.tsx#L515)